lib/src/bbox.dart

```diff
@@ -0,0 +1,24 @@
+import 'package:turf/coord_each.dart';
+import 'package:turf/helpers.dart';
+
+BBox bbox(GeoJSONObject geoJson) {
+  var result = [double.infinity, double.infinity, double.negativeInfinity, double.negativeInfinity];
+
+  coordEach(geoJson, (Position coord, _, __, ___, ____) {
+    print(coord);
+    if (result[0] > coord.lat) {
+      result[0] = coord.lat.toDouble();
```

Comment:
very important:
GeoJSON bounding boxes are stored in this order (!): [lng1, lat1(, alt1), lng2, lat2(, alt2)].

I think you have lat as the first argument, which is incorrect.

Reply:
Changed to return points in the correct order